pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -27,13 +27,21 @@ import (
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
 	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/logging"
 )
 
 const (
 	// manifests console plugin yaml directory location
 	routeRBACYamlDirectory  = "static/tekton-results/route-rbac"
 	internalDBYamlDirectory = "static/tekton-results/internal-db"
+	deploymentAPI           = "tekton-results-api"
```

Comment:
this does not seems to be used

Reply:
I think I removed it by mistake. It was used earlier. Let me re-add it in the condition.

Reply:
I have added it back in the condition.